libkbfs/block_retrieval_worker_test.go

```diff
@@ -236,7 +236,8 @@ func TestBlockRetrievalWorkerShutdown(t *testing.T) {
 
 	w.Shutdown()
 	block := &FileBlock{}
-	ch := q.Request(context.Background(), 1, nil, ptr1, block)
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
it's used to ensure that the context goroutine is broken out of its loop, so we don't leak goroutines out of this test.